draft-wing-cidfi.md

```diff
@@ -1230,12 +1230,12 @@ depletion, and suchlike.
 
 # API Integration for QUIC Stream and Packet-Level Prioritization
 
-For each QUIC or DTLS stream requiring differentiated service, the QUIC stack can
+For each QUIC or DTLS stream requiring differentiated service, the QUIC or DTLS stack can
 map that stream to a different Destination CID. The application-level code
-would require an API to instruct the QUIC stack that a particular stream
+would require an API to instruct the QUIC or DTLS stack that a particular stream
```

Comment:
I don't think it is a new API in case of DTLS, see https://datatracker.ietf.org/doc/rfc9146/. RFC9146 only allows a single CID and allows it be changed using a post-handshake messages. If we would like to use multiple CIDs with a DTLS session, we will require a new spec updating RFC9146.